os/mm/mm_heap/mm_manage_allocfail.c

```diff
@@ -21,8 +21,27 @@
  ****************************************************************************/
 
 #include <tinyara/config.h>
-#if !defined(__KERNEL__)
+
+#undef  CONFIG_DEBUG
+#undef  CONFIG_DEBUG_ERROR
+#undef  CONFIG_DEBUG_WARN
+#undef  CONFIG_DEBUG_VERBOSE
+#undef  CONFIG_LOGM
+#undef  CONFIG_DEBUG_MM
+#undef  CONFIG_DEBUG_MM_ERROR
+#define CONFIG_DEBUG 1
+#define CONFIG_DEBUG_ERROR 1
+#define CONFIG_DEBUG_WARN 1
+#define CONFIG_DEBUG_VERBOSE 1
```

Comment:
Because we only use lldbg here, no need to define WARN and VERBOSE. No problem to keep them, but unnecessary.